src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 14
+#define SOF_ABI_MINOR 16
```

Comment:
ABI 15 is open now, if kernel PR is approved we could make both 15.

Reply:
Changed to ABI 15, for both kernel and FW PRs